vllm/engine/arg_utils.py

```diff
@@ -456,7 +457,14 @@ def from_cli_args(cls, args: argparse.Namespace) -> 'EngineArgs':
         # Get the list of attributes of this dataclass.
         attrs = [attr.name for attr in dataclasses.fields(cls)]
         # Set the attributes from the parsed arguments.
-        engine_args = cls(**{attr: getattr(args, attr) for attr in attrs})
+        engine_args = cls(**{
+            attr: getattr(args, attr)
+            for attr in attrs if hasattr(args, attr)
+        })
+        if hasattr(args, "served_model_name"):
+            engine_args.served_model_name = args.served_model_name
```

Comment:
It is a multi-valued arg and will be a list
```suggestion
            engine_args.served_model_name = args.served_model_name[0]
```